res/values-th/strings.xml

```diff
@@ -1084,10 +1084,10 @@
     <string name="CIRCLE">วงกลม</string>
     <string name="SQUARES">สี่เหลี่ยม</string>
     <string name="TRICK_MODE">ทริคโหมด</string>
-    <string name="ICE_SPLIT">ICE SPLIT</string>
-    <string name="TRICK_SPLIT">TRICK SPLIT</string>
-    <string name="POP_SPLIT">POP SPLIT</string>
-    <string name="PUSH_SPLIT">PUSH SPLIT</string>
+    <string name="ICE_SPLIT">น้ําแข็งแยก</string>
+    <string name="TRICK_SPLIT">เคล็ดลับแยก</string>
+    <string name="POP_SPLIT">ป๊อปแยก</string>
+    <string name="PUSH_SPLIT">ผลักดันแยก</string>
```

Comment:
From lines 1087 to 1090:

That English text is like the previous one in all languages I think

Reply:
Yep so dont Change mod names again?

Reply:
They're used to those names, unless if the compehension ain't going to be affected in any way. I advise you to read **Tips** in the main repository page ^^